pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -372,5 +373,30 @@ func (ks *scaler) scale(ctx context.Context, pa *autoscalingv1alpha1.PodAutoscal
 	}
 
 	logger.Infof("Scaling from %d to %d", currentScale, desiredScale)
-	return desiredScale, ks.applyScale(ctx, pa, desiredScale, ps)
```

Comment:
It seems you still have `func (ks *scaler) applyScale(ctx context.Context...` left unused.

Reply:
Yes, it can be delete.